packages/cf-component-page/src/PageHeader.js

```diff
@@ -6,7 +6,7 @@ class PageHeader extends React.Component {
       <header className="cf-page__header">
         <h1 className="cf-page__title">{this.props.title}</h1>
         {this.props.subtitle &&
-          <p className="cf-page__subtitle">{this.props.subtitle}</p>}
+          <h2 className="cf-page__subtitle">{this.props.subtitle}</h2>}
```

Comment:
This is actually the right approach. There's only one kind of PageHeader, and the default styles in cf-component-heading are not contextual. `<Heading size={2}>` can mean very different things in different circumstances. There's no one size fits all approach here, as evident by the `<h1>` above.